FrameRate/FrameRate.cpp

```diff
@@ -381,7 +380,7 @@ namespace WPEFramework
                 int maxFps = -1;
                 if (m_numberOfFpsUpdates > 0)
                 {
-                averageFps = (m_totalFpsValues / m_numberOfFpsUpdates);
+                averageFps = (m_totalFpsValues / 2);
```

Comment:
Checking why this is changed to 2. averageFps is meant to return (m_totalFpsValues / m_numberOfFpsUpdates)